opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -1958,6 +1992,16 @@ const completeSpecialFilterKeys = async (context, user, inputFilters) => {
       if (newFilterGroup) {
         finalFilterGroups.push(newFilterGroup);
       }
+    } else if (arrayKeys.includes(WORKFLOW_FILTER) || arrayKeys.includes(X_OPENCTI_WORKFLOW_ID)) {
+      // in case we want to filter by status template (template of a workflow status) or status
+      // we need to find all statuses filtered by status template and filter on these statuses
+      const { newFilter, newFilterGroup } = await adaptFilterToWorkflowFilterKey(context, user, filter);
+      if (newFilter) {
+        finalFilters.push(newFilter);
+      }
+      if (newFilterGroup) {
+        finalFilterGroups.push(newFilterGroup);
+      }
```

Comment:
this would never happen, right ? We  can just return a newFilter and use it

Reply:
fixed